### ISSUE TITLE
Add TEST pools to testnet UI

### ISSUE THREAD

**Issue by just-a-node** [connext/monorepo#3933]
## Problem

TEST will be set up with pools and they need to be added to the UI.

**Comment by pjenviri** [comment]
Yes, we can do this. Please provide us the token addresses of both `local` and `adopt` on each chain when all of them are available. Thanks.

**Comment by just-a-node** [comment]
Revisit after migration to Sepolia.
